components/automate-ui/src/app/pages/policy/details/policy-details.component.ts

```diff
@@ -63,7 +68,14 @@ export class PolicyDetailsComponent implements OnInit, OnDestroy {
         this.policyJSON = this.policyToString(this.policy);
         const members = [];
         this.policy.members.forEach(element => {
-          members.push(stringToMember(element));
+          const member = stringToMember(element);
+          members.push(member);
+          if (member.type === Type.LocalUser) {
+            this.memberURLs[member.name] = ['/settings', 'users', member.displayName];
```

Comment:
Should member.name be changed to member.id ?

Reply:
Not for now, because of how the Member type is defined. I think we should refactor Member's `name` field to be `id` everywhere, but let's do that in a refactor PR and not mix that in with the bug fix imo :)